src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransportFactory.php

```diff
@@ -22,7 +22,15 @@ final class EsmtpTransportFactory extends AbstractTransportFactory
 {
     public function create(Dsn $dsn): TransportInterface
     {
-        $tls = 'smtps' === $dsn->getScheme() ? true : null;
+        $tls = null;
```

Comment:
this `null` will never be kept, as there is only those 2 supported schemes

Reply:
In `src/Symfony/Component/Mailer/Tests/TransportTest.php` there are cases like `dummy://` which will be resolved to either `smtp` or `smtps` internally.

Reply:
see `getSupportedSchemes`

Reply:
@stof see Daniel's question